src/components/common/atoms/ActionButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react';
+import { StyledIcon, Wrapper, StyledCount } from './styled';
+
+type Props = {
+  counter?: number | string;
+  icon: ReactNode;
+  onPress: () => void;
+};
+
+const ActionButton = ({ counter, icon, onPress }: Props) => {
+  return (
+    <Wrapper>
+      <StyledIcon onClick={onPress}>{icon}</StyledIcon>
+      <StyledCount>{!counter ? 0 : counter}</StyledCount>
```

Comment:
`counter ?? 0`

Checkout this out: [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

Reply:
did this I WIN I ALWAYS WIN